src/main/java/com/example/botjournal/config/BotInitializer.java

```diff
@@ -0,0 +1,30 @@
+package com.example.botjournal.config;
+
+import com.example.botjournal.sevices.TelegramBotService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import org.telegram.telegrambots.meta.TelegramBotsApi;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+import org.telegram.telegrambots.updatesreceivers.DefaultBotSession;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class BotInitializer {
+    private final TelegramBotService telegramBotService;
+
+    @EventListener({ContextRefreshedEvent.class})
+    public void init() throws TelegramApiException {
+        TelegramBotsApi telegramBotsApi = new TelegramBotsApi(DefaultBotSession.class);
+
+        try {
+            telegramBotsApi.registerBot(telegramBotService);
+        } catch (TelegramApiException e) {
+            log.error(e.getMessage());
```

Comment:
`log.error("Exception during bot registration", e);` Otherwise the part of stacktrace is missed which might involve more time in debugging the problem.